docs/user_guide/specifying_finance_parameters.md

```diff
@@ -0,0 +1,160 @@
+# Finance Parameters
+
+Finance parameters are primarily specified in the `plant_config` under the `finance_parameters` section. Some parameters from the `plant` section of the `plant_config` may also be used in finance calculations, these parameters are `plant_life`, `installation_time`, `atb_year`, and `cost_year`.
+
+There are two methods for specifying other finance parameters:
+- [Directly in finance parameters](finance:direct_opt)
+- [ProFAST parameters config](finance:pf_params_opt)
+
+Both methods require the following additional finance parameters to be specified:
+- `costing_generation_inflation` is used to adjust costs for each technology from the year provided under `discount_years` to the `cost_year` specified in the `plant_config` under the `plant` section.
+- `depreciation_method`: depreciation method to apply to capital items
+- `depreciation_period`: depreciation period (in years) for capital items (except electrolyzer, if used)
+- `depreciation_period_electrolyzer`: depreciation period (in years) for electrolyzer capital item (if used)
+
+(finance:direct_opt)=
+## Providing Finance Parameters: Direct Method
+Below is an example of the `finance_parameters` section of `plant_config` if financial parameters are input directly under the `finance_parameters` section.
+
+```yaml
+finance_parameters:
+  # Inflation parameters
+  profast_general_inflation: 0.0 # 0 for nominal analysis
+  costing_general_inflation: 0.025
+  # Finance parameters
+  discount_rate: 0.09
+  debt_equity_split: False
```

Comment:
I'm pretty sure only one of `debt_equity_split` or `debt_equity_ratio` should be included

Reply:
Oh, you are right. Thanks. We should fix that, but not necessarily in this PR